pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_user-roles-in-pantheon.adoc

```diff
@@ -0,0 +1,20 @@
+[id="user-roles-in-pantheon_{context}"]
+= User roles in {ProductShortName}
+
+In {ProductShortName}, each user is assigned a role that determines the actions the user can perform.
+
+There are two user roles defined in {ProductShortName} - `Author` and `Publisher`. The following permissions are associated with the roles:
```

Comment:
I just checked RHV documentation, and I see that the roles there are in bold. In any case, code formatting is not the right markup in this context. I suggest marking up the roles like so: **Author** and **Publisher**.

Reply:
done